### ISSUE TITLE
Cannot open Campaign 15 Target Pixel FITS files

### ISSUE THREAD

**Issue by StuartLittlefair** [KeplerGO/lightkurve#88]
After installing lightkurve on Anaconda python 3.6 (```pip install lightkurve```) and downloading a TPF FITS file from MAST (https://archive.stsci.edu/pub/k2/target_pixel_files/c15/249900000/14000/ktwo249914869-c15_spd-targ.fits.gz), I am unable to open with lightkurve

```
tpf = KeplerTargetPixelFile.from_fits_images(['ktwo249914869-c15_spd-targ.fits'])
...
FileNotFoundError: [Errno 2] No such file or directory: '/Users/sl/anaconda3/lib/python3.6/site-packages/lightkurve/data/tpf-ext0-header.txt'
```

I tried downloading ```tpf-ext0-header.txt``` etc, from ```kadenza``` after which the above command works, but when I produce a lightcurve it is empty...

**Comment by christinahedges** [comment]
@StuartLittlefair the from_fits_images also requires a position and size argument. If you don't pass a position in the image you would like to cut out, it will return something blank. You can read more here:

http://lightkurve.keplerscience.org/api/targetpixelfile.html#lightkurve.targetpixelfile.KeplerTargetPixelFile.from_fits_images

You can use astropy.SkyCoord to pass an RA and Dec to cut out with this tool

@barentsen it sounds like these files aren't being tracked by pip? https://github.com/KeplerGO/lightkurve/tree/master/lightkurve/data

**Comment by barentsen** [comment]
@StuartLittlefair Note that you are trying to open a Target Pixel File, so you want to execute `KeplerTargetPixelFile('ktwo249914869-c15_spd-targ.fits')`, not  `KeplerTargetPixelFile.from_fits_images('ktwo249914869-c15_spd-targ.fits')`

(Nevertheless: this does expose a bug in `from_fits_images`, and it also shows that we should print a warning if someone tries to open a TPF using `from_fits_images`.)

**Comment by StuartLittlefair** [comment]
Many thanks @barentsen and @christinahedges for the help, especially since this was a pretty basic user error on my part. 

@barentsen would you like me to raise a separate issue for the pip packaging problem, or leave this one open?

**Comment by barentsen** [comment]
Let's leave the ticket open -- I expect to resolve this today!

**Comment by barentsen** [comment]
The `FileNotFoundError` issue was resolved by adding a `MANIFEST.in` file to the repository.  I made this a hotfix and released `1.0beta4` to pypi.

I will close this issue and open a separate one to have `from_fits_images` print a helpful warning if someone tries to use it to open TPFs, which is not what it's intended for.

@StuartLittlefair Many many thanks for the bug report!
